fhir-operation-apply/src/main/java/com/ibm/fhir/operation/apply/ApplyOperation.java

```diff
@@ -138,8 +136,7 @@ protected Parameters doInvoke(FHIROperationContext operationContext,
 
     // Process the Parameters bodies for the contents
     private CodeableConcept checkAndProcessSettingContext(FHIROperationContext operationContext,
-        MultivaluedMap<String, String> queryParameters, Parameters parameters)
-        throws FHIROperationException {
+        MultivaluedMap<String, String> queryParameters, Parameters parameters) throws FHIROperationException {
```

Comment:
Just a minor style comment, but I'd prefer to see this line indented one more level so that its clear that its a continuation of the method declaration and make it stand out from the first line in the method impl.
```suggestion
            MultivaluedMap<String, String> queryParameters, Parameters parameters) throws FHIROperationException {
```